### PR TITLE
Jwoo/ios custom rendering

### PR DESCRIPTION
**Pull request by nesalang**
extend custom rendering to all objects

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseCardElement.mm [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    std::shared_ptr<BaseCardElement> _elem;
+}
+
+- (instancetype)init
+{
+    self = [super init];
+    return self;
+}
+
+- (std::shared_ptr<BaseCardElement>)getElem
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseCardElement.mm]: >getElem [](start = 36, length = 7)

Minor, but accessors in Objective-C aren't normally prefixed with `get`. The method name would just be `elem`, or even better, `element`.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigPrivate.h [hunk 2]
```diff
@@ -11,6 +11,7 @@ using namespace AdaptiveCards;
 
 @interface ACOHostConfig()
 
+- (instancetype)init:(std::shared_ptr<HostConfig> const &)config;
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigPrivate.h]: >init: [](start = 16, length = 5)

Also minor: initializers that take arguments are usually named initWithSomething. In this case, `initWithConfig`.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseCardElement.mm (hunk 1 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseCardElement.mm]: thank you

---
In reply to: [169506601](https://github.com/Microsoft/AdaptiveCards/pull/1143#discussion_r169506601) [](ancestors = 169506601)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigPrivate.h (hunk 2 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigPrivate.h]: thank you

---
In reply to: [169506801](https://github.com/Microsoft/AdaptiveCards/pull/1143#discussion_r169506801) [](ancestors = 169506801)

**Comment by Assaya** [comment]
Yeah!
Have a week off my friend x

GLOBALCHERRY

**Comment by Assaya** [comment]
https://open.spotify.com/album/17nNvZj3MTfCYuQ5j7H1fj?si=WG6yeTfkQyqzF0sk9GmIhA


angela@thehelm

[cid:d60aa2e4-d99f-4832-a18f-2101a22b88b7]

**Pull request merged**